README.md

```diff
@@ -21,7 +21,7 @@ Open MCT imposes on developers, such as the current requirement to use
 AngularJS.
 
 We want Open MCT to be as easy to use, install, run, and develop for as
-possible, and your feedback will help us get there! Feedback can be provided via [GitHub issues](https://github.com/nasa/openmct/issues), or by emailing us at [arc-dl-openmct@nasa.gov](mailto:arc-dl-openmct@nasa.gov).
+possible, and your feedback will help us get there! Feedback can be provided via [GitHub issues](https://github.com/nasa/openmct/issues), or by emailing us at [arc-dl-openmct@nasa.gov](mailto:arc-dl-openmct@mail.nasa.gov).
```

Comment:
We should probably update the displayed email address as well.

Reply:
@akhenry GOOD POINT!  Fixed.